run-kolibrihome.mount

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Kolibri data from Endless Live USB Storage
+ConditionKernelCommandLine=endless.image.device
+ConditionKernelCommandLine=endless.live_boot
+ConditionDirectoryNotEmpty=/run/media/eoslive/.kolibri
+
+[Mount]
+What=/run/media/eoslive/.kolibri
+Where=/run/kolibrihome
+Options=bind
```

Comment:
We need this to be a readable / writable directory, at least for desktop users running the Kolibri app. (This is a bit gross for a multi-user system so we have a much better system usually, but in this case it's a bit of a weird hack). My initial thought is to add `DirectoryMode=777`. Do you think that would be okay here?

Reply:
In my tests, `DirectoryMode` ends up not really having any effect since the `What` of this unit is backed by an exFAT filesystem, which does not have access control information. But yeah, in theory we would need that if the backing fs was any linux filesystem.

Reply:
Ah, got it. Thanks :)